index.js

```diff
@@ -31,6 +31,14 @@ var isPlainObject = function isPlainObject(obj) {
 	return typeof key === 'undefined' || hasOwn.call(obj, key);
 };
 
+var isBuffer = function isBuffer(buff) {
+	if (typeof Buffer === 'function' && typeof Buffer.isBuffer === 'function') {
```

Comment:
if we're going to call this, it should be cached in a closure so someone can't later change it